NtupleProducer/plugins/HTauTauNtuplizer.cc

```diff
@@ -1883,6 +1901,14 @@ void HTauTauNtuplizer::beginJob(){
 
   myTree->Branch("ak8jets_nsubjets", &_ak8jets_nsubjets);
 
+  myTree->Branch("ak8jets_chEmEF" , &_ak8jets_chEmEF);
+  myTree->Branch("ak8jets_chHEF"  , &_ak8jets_chHEF);
+  myTree->Branch("ak8jets_nEmEF"  , &_ak8jets_nEmEF);
+  myTree->Branch("ak8jets_nHEF"   , &_ak8jets_nHEF);
+  myTree->Branch("ak8jets_MUF"    , &_ak8jets_MUF);
+  myTree->Branch("ak8jets_neMult" , &_ak8jets_neMult);
+  myTree->Branch("ak8jets_chMult" , &_ak8jets_chMult);
```

Comment:
Are we using these branches in the skimming step? Do we we really need them?

Reply:
No we're not using them so far, but to be sure I thought I might as well add them... for the HEM issue (and maybe also jet veto maps?) we needed some of them for AK4 jets, but I can also remove them again... anyways so far accessing them for AK8 jets doesn't work anyway so I'll have to investigate further